Apps/Admin/Client/Pages/DashboardPage.razor

```diff
@@ -223,32 +223,32 @@ else if (YearOfBirthCountsErrorMessage.Length == 0)
                   data-testid="daily-data-table">
             <HeaderContent>
                 <MudTh>
-                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.DailyDateTime)"
+                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.Date)"
                                        InitialDirection="@SortDirection.Descending">
                         Date
                     </MudTableSortLabel>
                 </MudTh>
                 <MudTh>
-                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.TotalRegisteredUsers)">
+                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.UserRegistrations)">
                         Registered
                     </MudTableSortLabel>
                 </MudTh>
                 <MudTh>
-                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.TotalLoggedInUsers)">
+                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.UniqueLogins)">
                         Logged In
                     </MudTableSortLabel>
                 </MudTh>
                 <MudTh>
-                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.TotalDependents)">
+                    <MudTableSortLabel SortBy="new Func<DailyDataRow, object>(x => x.DependentRegistrations)">
                         Dependents
                     </MudTableSortLabel>
                 </MudTh>
             </HeaderContent>
             <RowTemplate>
-                <MudTd data-testid="daily-data-date" DataLabel="Date">@context.DailyDateTime.ToString("yyyy-MM-dd")</MudTd>
-                <MudTd data-testid="daily-data-total-registered-users" DataLabel="Registered">@context.TotalRegisteredUsers</MudTd>
-                <MudTd data-testid="daily-data-total-logged-in-users" DataLabel="Logged In">@context.TotalLoggedInUsers</MudTd>
-                <MudTd data-testid="daily-data-dependents" DataLabel="Dependents">@context.TotalDependents</MudTd>
+                <MudTd data-testid="daily-data-date" DataLabel="Date">@context.Date.ToString("yyyy-MM-dd")</MudTd>
+                <MudTd data-testid="daily-data-total-registered-users" DataLabel="Registered">@context.UserRegistrations</MudTd>
+                <MudTd data-testid="daily-data-total-logged-in-users" DataLabel="Logged In">@context.UniqueLogins</MudTd>
+                <MudTd data-testid="daily-data-dependents" DataLabel="Dependents">@context.DependentRegistrations</MudTd>
```

Comment:
*NIT:*
The dropping of the `Total*` removes the clarity that its count, don't see the need for this change

Reply:
It was renamed because it was confusing in another way - these are calculated by a Sum() that isn't actually adding multiple values.  If you think it provides value, they could be renamed to UserRegistrationCount / UniqueLoginCount / DependentRegistrationCount.